src/main/resources/mcmod.info

```diff
@@ -3,8 +3,8 @@
   "modid": "blackbox",
   "name": "Blackbox Rendering API",
   "description": "A complex system or device whose internal workings are hidden or not readily understood.",
-  "version": "${version}",
-  "mcversion": "${mcversion}",
+  "version": "5",
```

Comment:
why

Reply:
This is why

https://mclo.gs/yW35tqf

Reply:
so in this case, why hardcode it when there's already a matched property named `mod_version` ?

Reply:
because i got lazy, its fixed now